nilearn/plotting/data/html/report_body_template.html

```diff
@@ -1,85 +1,81 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
-
-<head>
-<title>HTML report</title>
-<meta name = "viewport" content = "width = device-width, initial-scale = 1">
+<!-- CSS for the report -->
 <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css" integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
+<link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/grids-responsive-min.css">
 <style type="text/css">
 
-h1 { padding-top: 15px; }
+div.nilearn_report {
+    padding-top: 5px;
+}
 
-body {
-    padding: 10px 20px 10px 20px;
+div.nilearn_report h1 {
+    padding-top: 15px;
 }
 
-summary::-webkit-details-marker {
-margin-right: 2px;
+div.nilearn_report summary::-webkit-details-marker {
+    margin-right: 2px;
 }
 
-summary:focus {
- outline-style: none;
+div.nilearn_report summary:focus {
+    outline-style: none;
 }
 
-table {
+div.nilearn_report table {
     max-width: 100%;
     border-collapse: collapse;
     margin:50px auto;
 }
 
-thead th {
+div.nilearn_report thead th {
   text-align: center;
 }
 
-tbody tr:nth-child(even) {
+div.nilearn_report tbody tr:nth-child(even) {
    background-color: #ddd;
 }
 
 /* Tooltip container */
-.tooltip {
+div.nilearn_report .withtooltip {
```

Comment:
"tooltip" -> "withtooltip" is because jupyter already uses "tooltip" in their CSS.